src/applications/personalization/dashboard/components/Dashboard.jsx

```diff
@@ -50,7 +50,7 @@ import CTALink from './CTALink';
 import BenefitPaymentsAndDebt from './benefit-payments-and-debts/BenefitPaymentsAndDebt';
 import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
 import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
-import DismissibleDebts from './benefit-payments-and-debts/DismissibleDebts';
+import DebtNotification from './notifications/DebtNotification';
```

Comment:
Should the component in `DebtNotification.jsx` be renamed from `Debts` to `DebtNotification`?

Reply:
I do agree that I should change it for best practices